pom.xml

```diff
@@ -130,16 +82,16 @@
 
     <dependencyManagement>
         <dependencies>
-            <!-- hugegraph-commons -->
+            <!-- TODO: could we merge them to hugegraph-commons only? -->
```

Comment:
no we can't merge them, because in some scenarios it is not necessary to import both the two packages at the same time, like client or loader.

Reply:
OK, change the comment in next PR (could modify the common version together @z7658329 )